python/python/lance/tf/data.py

```diff
@@ -265,5 +266,113 @@ def lance_fragments(dataset: Union[str, Path, LanceDataset]) -> tf.data.Dataset:
     )
 
 
+def _ith_batch(i: int, batch_size: int, total_size: int) -> Tuple[int, int]:
+    start = i * batch_size
+    end = tf.math.minimum(start + batch_size, total_size)
+    return (start, end)
+
+
+def lance_batches(
```

Comment:
does `tf.data` has a `from_batches`, can we name this to `from_batch` to be more consist with the tf style

Reply:
Sure